tests/openvino/test_modeling.py

```diff
@@ -1219,24 +1219,24 @@ class OVModelForCausalLMIntegrationTest(unittest.TestCase):
         "gpt_bigcode": 5,
         "blenderbot": 2,
         "blenderbot-small": 2,
-        "bloom": 5,
+        "bloom": 5 if is_transformers_version(">=", "4.44") else 0,
         "chatglm": 2,
         "codegen": 5,
         "codegen2": 2,
-        "gpt2": 5,
-        "gptj": 5,
-        "gpt_neo": 4,
-        "gpt_neox": 5,
+        "gpt2": 5 if is_transformers_version(">=", "4.42") else 0,
+        "gptj": 5 if is_transformers_version(">=", "4.42") else 0,
+        "gpt_neo": 4 if is_transformers_version(">=", "4.42") else 0,
+        "gpt_neox": 5 if is_transformers_version(">=", "4.42") else 0,
```

Comment:
coming from optimum bettertransformers depreciation https://github.com/huggingface/optimum/pull/2305

Reply:
4.42 or 4.45 might be a good new min version

Reply:
yes agreed! upgraded to 4.45